external-crates/move/crates/move-package/src/compilation/versioned_compilation.rs

```diff
@@ -177,6 +186,52 @@ fn download_and_compile<W: Write>(
     Ok(())
 }
 
+pub fn decode_bytecode_file(
```

Comment:
This is basically a copy pasta of `OnDiskCompiledPackage::decode_unit`. I didn't want to instantiate `OnDiskCompiledPackage` just to use this function (annoying). I can factor it out so there is a shared free function, but I wasn't convinced that was useful either.